addons/editor-theme3/theme3.css

```diff
@@ -29,3 +29,31 @@
   fill: var(--editorTheme3-commentText);
   color: var(--editorTheme3-commentText);
 }
+
+.sa-colormode-submenu li:nth-child(1) div,
+.sa-colormode-submenu li:nth-child(2) div {
+  filter: opacity(0.5) grayscale(1);
+}
+.sa-colormode-submenu li:nth-child(1) div img:first-child,
+.sa-colormode-submenu li:nth-child(2) div img:first-child {
+  visibility: hidden;
+}
+.sa-theme3-link div img:first-child {
+  display: none;
+}
+.sa-colormode-submenu li:nth-child(1):hover,
+.sa-colormode-submenu li:nth-child(2):hover,
+.sa-theme3-managed:hover {
+  background-color: inherit;
+  cursor: default;
+}
+.sa-theme3-managed {
+  border-bottom: 1px solid white;
```

Comment:
The white border will be invisible if the menu bar background is set to white. A better way to add a separator is to add Scratch's `menu_menu-section` class to the "Addon Settings" item.